app/models/check.rb

```diff
@@ -0,0 +1,18 @@
+class Check < ApplicationRecord
+  belongs_to :claim
+
+  enum result: {
+    approved: 0,
+  }
+
+  validates :result, presence: true
+  validates :checked_by, presence: true
+
+  before_create do
+    self.checked_at = Time.zone.now
+  end
```

Comment:
Went for this because the default wasn't working.

Reply:
Is that short hand for a `before_save` lifecycle hook?

Reply:
`before_create` is a different callback from `before_save`, and you can pass a block in instead of the name of a method (not sure which aspect you were asking about).